opslevel/resource_opslevel_check_repository_search.go

```diff
@@ -102,7 +115,7 @@ func (r *CheckRepositorySearchResource) Schema(ctx context.Context, req resource
 func (r *CheckRepositorySearchResource) ValidateConfig(ctx context.Context, req resource.ValidateConfigRequest, resp *resource.ValidateConfigResponse) {
 	var configModel CheckRepositorySearchResourceModel
 	resp.Diagnostics.Append(req.Config.Get(ctx, &configModel)...)
-	if resp.Diagnostics.HasError() {
+	if resp.Diagnostics.HasError() || configModel.FileContentsPredicate.Type.IsUnknown() {
```

Comment:
Missing nil check

Reply:
This nil check is not need here. See [above comment](https://github.com/OpsLevel/terraform-provider-opslevel/pull/361#discussion_r1626314117)